src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendRecorder.java

```diff
@@ -91,6 +91,10 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, final Build
                     filterForm.setVulnTypes(Collections.singletonList(condition.getThresholdVulnType()));
                 }
 
+                if (!profile.getVulnerabilityStatuses().isEmpty()) {
```

Comment:
Let's add a test around this. Write the test in such a way that the test won't change much if we refactor how the profile handles statuses.

Test that: 
* Given there are statuses selected, those statuses get added to the filter form
* If there not no statuses, nothing is added to filter form.

Reply:
Tests are added.